src/App.tsx

```diff
@@ -12,50 +14,93 @@ function App() {
   const [isReversed, setIsReversed] = useState<boolean>(false)
   const [leftCurrency, setLeftCurrency] = useState<string>('')
   const [rightCurrency, setRightCurrency] = useState<string>('')
+  const [exchangeRate, setExchangeRate] = useState<string|number>('')
+  const [allCurrenciesSymbols, setAllCurrenciesSymbols] = useState<Symbol[]>([])
+  const [leftSymbol, setLeftSymbol] = useState<string|undefined>()
+  const [rightSymbol, setRightSymbol] = useState<string|undefined>()
+
+  useEffect(() => {
+    addSymbolToInput({leftCurrency, rightCurrency, allCurrenciesSymbols, setLeftSymbol, setRightSymbol})
+  },[allCurrenciesSymbols, leftCurrency, rightCurrency])
 
   useEffect(() => {
-    const getCurrencies = async () => {
+    const setSymbolsAndAllCurrencies = async () => {
```

Comment:
if we will add a new API request(get a user for example), how you will name this function?

Reply:
apiRequests or gettingDate

Reply:
gettingData*